TemplateBuilder/src/assets/styles/template-builder.scss

```diff
@@ -0,0 +1,62 @@
+.spinner-container {
+  height: 240px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+:host {
+  display: block;
+  padding: 0.75rem;
+  overflow-x: auto;
+  min-width: 1368px;
+  color: var(--md-textColor-secondary);
```

Comment:
this seems like an random color to choose. also, provide fallback color.

Reply:
Addressed